src/app/app.module.tns.ts

```diff
@@ -30,4 +32,9 @@ import {IntroComponent} from './intro/intro.component';
   bootstrap: [AppComponent],
   schemas: [NO_ERRORS_SCHEMA]
 })
-export class AppModule { }
+export class AppModule {
+  constructor() {
+    platformCss.sizeGroupings(true);
+    platformCss.sizeGroupings([9999,740,320]);
```

Comment:
Magic numbers ✨ 🎩  => put into named vars

Reply:
Yes, this will change, only here as a demo and it is not at all working as we want.

Reply:
Ok check